metricbeat/module/redis/info/info.go

```diff
@@ -43,45 +45,57 @@ func (m *MetricSeter) Setup(ms *helper.MetricSet) error {
 	}
 
 	for _, host := range ms.Config.Hosts {
+		redisPool := createPool(host, config.Password, config.Network, config.MaxConn, ms)
```

Comment:
I tend to prefer the more explicit params over passing a config map here. I meant to pass `ms.Module.Timeout` instead of the whole metricset as well.